src/views/datasources/list/DataSourcesList.tsx

```diff
@@ -35,12 +42,22 @@ const DataSourcesList: React.FC<DataSourcesListProps> = ({ kind, namespace }) =>
   const filters = getDataImportCronFilter(t);
   const [unfilteredData, data, onFilterChange] = useListPageFilter(dataSources, filters);
 
+  const createItems = {
+    form: t('With Form'),
+    yaml: t('With YAML'),
```

Comment:
Do we need also a YAML data source example? Like in template and vm?

Reply:
We should add an option to create by YAML yes